Tools/peg_generator/pegen/c_generator.py

```diff
@@ -37,6 +37,8 @@
 #  define D(x)
 #endif
 
+# define MAXSTACK 1700
```

Comment:
This was taken from the old stack limit:

https://github.com/python/cpython/blob/8202a7e3dd55d551906390742fc748d0747256b8/Parser/parser.h#L10